modules/ROOT/pages/configure-openshift-routes.adoc

```diff
@@ -81,16 +81,96 @@ You can configure the ingress resource template to include passthrough TLS termi
 For more information on configuring routes with passthrough TLS termination, refer to the https://docs.openshift.com/container-platform/4.10/networking/routes/route-configuration.html#nw-ingress-creating-a-route-via-an-ingress_route-configuration[Red Hat OpenShift documentation^].
 
 Follow these steps to create an ingress resource template:
+
 . In the template, add the following annotation:
 +
 ----
 route.openshift.io/termination: passthrough
 ----
+[start=2]
 . In the template, set the following parameter values:
 +
-* Set `path: ''` (an empty string) 
-* Set `pathType: ImplementationSpecific`
+* `path: ''` (an empty string) 
+* `pathType: ImplementationSpecific`
 
-. In Runtime Manager, when configuring ingress for a Mule application, in the *Path* field include a `/`.
+. In Runtime Manager, when configuring ingress for a Mule app, in the *Path* field include a `/`.
 
 The application endpoint will show HTTPS for passthrough TLS termination.
+
+== Configure Reencrypt Termination
+
+You can configure the ingress resource template to include reencrypt termination by adding the `route.openshift.io/termination: reencrypt` annotation to the template.
+
+Follow these steps to configure reencrypt termination:
+
+. Create a secret object to hold the destination CA certificate, which can be a self signed certificate, or chain. +
+The following mule-app.cert file contains a PEM-encoded certificate (chain or self-signed certificate) of the leaf certificate used in target Mule app's keystore.
++
+----
+kubectl -nrtf create secret generic ah-destination-tls \
+--from-file=tls.crt=mule-app.cert
+----
++
+[start=2]
+. Add a label to the secret to propagate it into the Mule app's namespace.
++
+----
+kubectl -n rtf label secret ah-destination-tls "rtf.mulesoft.com/synchronized=true""
+----
++
+[start=3]
+. Create or delete and re-create the ingress template in Runtime Fabric namespace with the additional `route.openshift.io/destination-ca-certificate-secret:` annotation.
++
+----
+> cat ingress-reencrypt.yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    route.openshift.io/termination: "reencrypt"
+    route.openshift.io/destination-ca-certificate-secret: "ah-destination-tls"
+  name: rtf-ingress
+  namespace: rtf
+  labels:
+spec:
+  ingressClassName: rtf-openshift
+  tls:
+  - hosts:
+      - app-name.andrew.com
+    secretName: ah-tls
```

Comment:
I am assuming that ah-tls secret will be created and configured same as it is done for other termination options.